docs/source/configuration/experimental.md

```diff
@@ -0,0 +1,35 @@
+---
+myst:
+  html_meta:
+    "description": "Enable experimental features in Volto's configuration object"
+    "property=og:description": "Enable experimental features in Volto's configuration object"
+    "property=og:title": "Experimental features"
+    "keywords": "Volto, Plone, frontend, experimental"
+---
+
+# Experimental features
+
+Experimental features are planned for inclusion in a future release of Volto,
+but are not yet considered mature by the community.
+
```

Comment:
```suggestion
An experimental feature would typically be released in a canary release, giving users an easy way to install it and see if anything breaks before we make a new stable release that includes it.

```

Reply:
@stevepiercy I don't think this really helps clarify. That's true of non-experimental features too.

Reply:
Maybe this?

```suggestion
An experimental feature gives users an easy way to install it and see if anything breaks before we make a new stable release that includes it.

```